SWLOR.Game.Server/Feature/QuestDefinition/HutlarQuestDefinition.cs

```diff
@@ -155,5 +156,42 @@ private void StupendousSlugBile()
                 .AddItemReward("slug_shake", 1);
         }
 
+        private void BreakTheByysk()
+        {
+            _builder.Create("break_the_byysk", "Break the Byysk")
+
+                .AddState()
+                .SetStateJournalText("Sharene wants you to kill One Hundred Byysk. Of you go!")
+                .AddKillObjective(NPCGroupType.Byysk_Guardian, 100)
+
+                .AddState()
+                .SetStateJournalText("Return to Sharene in the Hutlar Outpost and report your progress.")
+
+                .AddState()
+                .SetStateJournalText("Sharene wants you to kill another One Hundred Byysk. Better get going")
+                .AddKillObjective(NPCGroupType.Byysk_Guardian, 100)
+
+                .AddState()
+                .SetStateJournalText("Return to Sharene, once again, at the Hutlar Outpost.")
+
+                .AddState()
+                .SetStateJournalText("Sharene wants you to kill ANOTHER One Hundred Byysk...")
+                .AddKillObjective(NPCGroupType.Byysk_Guardian, 100)
+
+                .AddState()
+                .SetStateJournalText("Return to Sharene, AGAIN.")
+
+                .AddState()
+                .SetStateJournalText("Sharene wants you to kill more Byysk... Let's go for one thousand this time!")
+                .AddKillObjective(NPCGroupType.Byysk_Guardian, 100)
```

Comment:
Journal text references 1000, but only 100 is listed here.

Reply:
Disregard. I kept reading, lol

Reply:
Hope is that after doing it 3x, people won't look at the quest log and it will pop ;)